LDEventSource/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.3.2</string>
+	<string>3.0.0</string>
```

Comment:
We're not so good about updating this. I'll add it to the release checklist, but is there a way to refactor the setup so we don't repeat the version string in multiple places?